checksum/src/test/scala/uk/gov/nationalarchives/checksum/ChecksumCalculatorTest.scala

```diff
@@ -0,0 +1,101 @@
+package uk.gov.nationalarchives.checksum
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers._
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+import uk.gov.nationalarchives.checksum.utils.TestUtils._
+import io.circe.parser.decode
+import graphql.codegen.AddFileMetadata.addFileMetadata.AddFileMetadata
+
+import scala.util.Try
+
+class ChecksumCalculatorTest extends AnyFlatSpec with BeforeAndAfterAll with BeforeAndAfterEach  {
+
+  override def beforeAll(): Unit = {
+    s3Api.start
+    sqsApi.start()
+    outputQueueHelper.createQueue
+    inputQueueHelper.createQueue
+  }
+
+  override def beforeEach(): Unit = {
+    createBucket
+    outputQueueHelper.receive.foreach(outputQueueHelper.delete)
+    inputQueueHelper.receive.foreach(inputQueueHelper.delete)
+  }
+
+  override def afterEach(): Unit = {
+    deleteBucket()
+  }
+
+  "The method" should "put a message in the output queue if the message is successful " in {
```

Comment:
Should this be named something more descriptive than "The method"?

Reply:
Yep, I've changed it to "The update method"